pyfixest/did/visualize.py

```diff
@@ -72,26 +93,132 @@ def panelview(
         subsamp = 50,
         title = "Treatment Assignment"
     )
+
+    # Outcome plot
+    panelview(
+        data = df_het,
+        unit = "unit",
+        time = "year",
+        type = "outcome",
+        outcome = "dep_var",
+        treat = "treat",
+        subsamp = 50,
+        title = "Outcome Plot"
+    )
     ```
     """
-    treatment_quilt = data.pivot(index=unit, columns=time, values=treat)
-    treatment_quilt = treatment_quilt.sample(subsamp) if subsamp else treatment_quilt
-    if collapse_to_cohort:
-        treatment_quilt = treatment_quilt.drop_duplicates()
-    if sort_by_timing:
-        treatment_quilt = treatment_quilt.loc[
-            treatment_quilt.sum(axis=1).sort_values().index
-        ]
-    if not ax:
-        f, ax = plt.subplots()
-    cax = ax.matshow(treatment_quilt, cmap="viridis", aspect="auto")
-    f.colorbar(cax) if legend else None
-    ax.set_xlabel(xlab) if xlab else None
-    ax.set_ylabel(ylab) if ylab else None
+    if type == "outcome" and outcome:
+        if units_to_plot:
```

Comment:
For the `units_to_plot` argument, I would say that we should not allow subsampling and aggregation, right? Because if a user specifies specific units, does it make sense to still aggregate them / to subsample from them? (I think you could maybe argue for the first?). Still I would rather restrict this behavior.